impeller/aiks/paint.cc

```diff
@@ -121,6 +122,34 @@ std::shared_ptr<Contents> Paint::WithColorFilter(
                                               absorb_opacity);
 }
 
+std::shared_ptr<FilterContents> Paint::MaskBlurDescriptor::CreateMaskBlur(
```

Comment:
I would defer to @bdero to review the mask blur implementation. Overall code looks reasonable to me.